projects/pastanaga-angular/src/lib/avatar/avatar.component.html

```diff
@@ -1,12 +1,12 @@
-<span class="pa-avatar pa-avatar-{{size}}">
+<span class="pa-avatar pa-avatar-{{size}}" [class.no-avatar]="!initials">
     <img *ngIf="base64Image; else avatarInitials"
          [src]="base64Image"
          class="pa-avatar-img">
 
     <ng-template #avatarInitials>
-        <abbr [attr.title]="userName"
-              class="pa-avatar-initials {{backgroundColorClass}}"
-        >
+        <abbr *ngIf="!!initials || !!userId"
```

Comment:
I find it hard to understand the logic behind this if statement... 
as far as I understand the component, it display either an image, either the "userName"
I can't imagine a use case where it would display nothing...